packages/shared/components/MenuLogin/MenuLogin.tsx

```diff
@@ -124,11 +118,53 @@ export const LoginItemList = ({ logins, onClick, onKeyPress, placeholder }) => {
         placeholder={placeholder}
         autoComplete="off"
       />
-      {$menuItems}
+      {content}
     </Flex>
   );
 };
 
+function getLoginItemListContent(
+  getLoginItemsAttempt: Attempt<LoginItem[]>,
+  onClick: (e: React.MouseEvent<HTMLAnchorElement>, login: string) => void
+) {
+  switch (getLoginItemsAttempt.status) {
+    case '':
+    case 'processing':
+      return (
+        <Indicator
```

Comment:
I noticed a small delay between opening the menu and displaying the loading spinner which is caused by `Indicator` component. Maybe we would like to set `delay="none"` on it?

Reply:
I think that's how Indicator works on purpose: in good conditions you'll never see a spinner, because the usernames will load that fast. If it takes longer than expected, we show the spinner.

Reply:
I noticed this because `Indicator` creates its own "background". When the data takes a little longer to load, you will first see only the input row, then the height will change and you will see the spinner. It would look better if the height did not change. 
But it's a really minor thing and we can skip it.

https://user-images.githubusercontent.com/20583051/164685089-4f29b4de-6da2-4e89-9c4f-dcbe63cd1a2a.mov